packages/docusaurus-plugin-content-docs/src/__tests__/__fixtures__/versioned-sites-2/v-site-1/docusaurus.config.js

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
I don't really like the name `versioned-sites-2`: when browsing that doc, I have no idea why there is a "2" in the name and what's the purpose of this fixture.

Please make it explicit, maybe rename versioned-sites-2 to `versioned_docs_path_fixtures` or something? don't hesitate to nest the site + its associated content in a subfolder so that we understand this association better